src/Basque/Mews.Fiscalizations.Basque/Model/TbaiInvoiceData.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Mews.Fiscalizations.Basque.Model
+{
+    public sealed class TbaiInvoiceData
+    {
+        public TbaiInvoiceData(string tbaiIdentifier, string qrCodeUri)
+        {
+            TbaiIdentifier = tbaiIdentifier;
+            QrCodeUri = qrCodeUri;
+        }
+
+        public string TbaiIdentifier { get; }
+
+        public string QrCodeUri { get; }
```

Comment:
QrCodeUri holds the actual data? Is it still an URI then?

Reply:
Its actually a URI, it doesn't contain the QR data.